apps/console/src/features/applications/data/application-templates/templates/android-mobile-application/create-wizard-help.tsx

```diff
@@ -42,16 +42,16 @@ const AndroidApplicationCreateWizardHelp: FunctionComponent<AndroidApplicationCr
 
     return (
         <div data-testid={ testId }>
-            <Heading as="h5">Name to identify your application</Heading>
-            <p>A unique name for the application.</p>
+            <Heading as="h5">Name</Heading>
+            <p>A unique name to identify your application.</p>
             <p>E.g., My App</p>
 
             <Divider />
 
             <React.Fragment>
                 <Heading as="h5">Authorized redirect URIs</Heading>
                 <p>
-                    The redirect URL determines where the authorization code is sent to
+                    The redirect URLs determine where the authorization code is sent to
```

Comment:
`The redirect URLs determine` - > `The authorized redirect URLs determine`

Reply:
Fixed in new commit